diff --git a/node_modules/metro-cache/src/stableHash.js.flow b/node_modules/metro-cache/src/stableHash.js.flow
index f3ceb66..2dcfde7 100644
--- a/node_modules/metro-cache/src/stableHash.js.flow
+++ b/node_modules/metro-cache/src/stableHash.js.flow
@@ -16,7 +16,7 @@ const crypto = require('crypto');
 function stableHash(value: mixed): Buffer {
   return (
     crypto
-      .createHash('md4')
+      .createHash('md5')
       /* $FlowFixMe(>=0.95.0 site=react_native_fb) This comment suppresses an
        * error found when Flow v0.95 was deployed. To see the error, delete this
        * comment and run Flow. */
